### ISSUE TITLE
 `az signalr create` fails when resource already exists

### ISSUE THREAD

**Issue by michael-wolfenden** [Azure/azure-cli#9382]
**Describe the bug**
Running the following command:

```
az signalr create --name "some-name" --resource-group "some-resource-group" --sku "Free_DS2"
```

when the resource already exists results in the following error:

```
Operation failed with status: 200. Details: Resource state Succeeded
```

**To Reproduce**
Running the following command **twice**:

```
az signalr create --name "some-name" --resource-group "some-resource-group" --sku "Free_DS2"
```

**Expected behavior**
I expected the creation of a SignalR resource to be idempotent and so result in no error when run a second time. This would be consistent with the behavior of other azure resources (ie resource groups, cosmos db, maps)

**Environment summary**
```
Linux-4.9.125-linuxkit-x86_64-with
Python 3.6.4
Shell: bash

azure-cli 2.0.64
```

**Comment by sunib** [comment]
Is there any progress on this? I'm noticing the same behavior. Creating the resource works fine, but it's sadly enough not idempotent due to this behavior.

My output of `az -v`

```
azure-cli                         2.0.80

command-modules-nspkg              2.0.3
core                              2.0.80
nspkg                              3.0.4
telemetry                          1.0.4

Extensions:
azure-devops                      0.17.0

Python location 'C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\python.exe'
Extensions directory 'C:\Users\simon\.azure\cliextensions'

Python (Windows) 3.6.6 (v3.6.6:4cf1f54eb7, Jun 27 2018, 02:47:15) [MSC v.1900 32 bit (Intel)]
```

**Comment by pleinolijf** [comment]
Having exactly the same issue. Not blocking, but definitely annoying, since you have to set your pipeline up to not fail because of the false error "Operation failed with status: 200. Details: Resource state Succeeded".  So now each and every build is displayed with a warning, which always raises concern with other people.

**Comment by zackliu** [comment]
Already fixed in the current azure-cli version 2.9.1.

**Comment by SaurabhSharma-MSFT** [comment]
@michael-wolfenden Apologies for the delay. Please let me know if you still seeing any issues as this looks like fixed.
